components/dashboard/src/admin/License.tsx

```diff
@@ -0,0 +1,215 @@
+
+    const featureList = license?.enabledFeatures;
+    const features = license?.features;
+
+    // if user seats is 0, it means that there is user limit in the license
+    const userLimit = license?.seats == 0 ? "Unlimited" : license?.seats;
+
+    const [licenseLevel, paid, statusMessage] = license ? getSubscriptionLevel(license) : defaultMessage();
+
+    return (
+        <div>
+            <PageWithSubMenu
+                subMenu={adminMenu}
+                title="License"
+                subtitle="License associated with your Gitpod Installation"
+            >
+                <div className="flex flex-row space-x-4">
+                    <Card>
+                        {licenseLevel}
+                        {paid}
+                        <div className="mt-4 font-semibold">Available features:</div>
+                        <div className="flex flex-col items-start">
+                            {features &&
+                                features.map((feat: string) => (
+                                    <span className="inline-flex space-x-1">
+                                        {featureList?.includes(feat) ? (
+                                            <CheckSvg fill="currentColor" className="self-center" />
+                                        ) : (
+                                            <XSvg fill="currentColor" className="self-center h-2" />
+                                        )}
+                                        <span>{capitalizeInitials(feat)}</span>
+                                    </span>
+                                ))}
+                        </div>
+                    </Card>
+                    <SolidCard>
+                        <div className="text-gray-600 dark:text-gray-200 py-4 flex-row flex font-semibold items-center">
+                            {statusMessage}
+                        </div>
+                        <p className="dark:text-gray-500 font-semibold">Registered Users</p>
+                        <span className="dark:text-gray-300 pt-1 text-lg">{license?.userCount || 0}</span>
+                        <span className="dark:text-gray-500 text-gray-400 pt-1 text-lg"> / {userLimit} </span>
+                        <p className="dark:text-gray-500 pt-2 font-semibold">License Type</p>
+                        <h4 className="dark:text-gray-300 pt-1 text-lg">{capitalizeInitials(license?.type || "")}</h4>
+                        <div className="flex justify-end bottom-2 relative">
+                            <button
+                                type="button"
+                                onClick={(e) => {
+                                    e.preventDefault();
+                                    window.location.href = "https://www.gitpod.io/self-hosted";
+                                }}
+                            >
+                                <div className="font-semibold">Compare Plans</div>
+                            </button>
```

Comment:
@gtsiolis Don't we also have a link that looks like a button (`a` tag with `href` etc)? Since this is just a link to another page, in my opinion that would be preferable so that we can get rid of the `onClick` JavaScript event.

Advantages:
- Users can see the link target on hover.
- Users can open the page in a new tab.

Reply:
Related: https://github.com/gitpod-io/gitpod/pull/9343#discussion_r855126635

Reply:
Thanks for noticing, @corneliusludmann! The intent of using a primary (confirm) action button here was to provide at least one primary action users can take. However, since there is no self-serve flow (yet) where users can request or manage their license, going with a link sounds good.

We could keep the button as is and introduce an appending icon to indicate the external link or simply go with a plain `<a>` link. Hint: Using the `gp-link` class could help as we need a different blue color for the light and dark theme.

| Button | Link (Light) | Link (Dark) |
|-|-|-|
| <img width="1440" alt="Community" src="https://user-images.githubusercontent.com/120486/164464479-d0307851-3841-4757-a106-521fbc747bdc.png"> | <img width="1440" alt="Community" src="https://user-images.githubusercontent.com/120486/164464891-8293601e-bb2d-470b-8a7d-6f526fce8b1e.png"> | <img width="1440" alt="Community-1" src="https://user-images.githubusercontent.com/120486/164464896-406c2b30-620a-41f8-8b92-b72887139874.png"> |


I've also updated the specs linked in https://github.com/gitpod-io/gitpod/issues/8413#issuecomment-1089213673 to include these new designs. Cc @nandajavarma